app/src/main/java/ch/epfl/sdp/drone3d/RegisterActivity.kt

```diff
@@ -35,8 +35,7 @@ class RegisterActivity : AuthActivity() {
     override fun success() {
         // Registration is a success, show Toast and main activity
         startActivity(Intent(this, MainActivity::class.java))
-        Toast.makeText(baseContext, R.string.register_success,
-                Toast.LENGTH_SHORT).show()
+        ToastHandler.showToast(baseContext, R.string.register_success)
```

Comment:
Should we really display a toast for this ? As a user of the app I would find it pretty annoying

Reply:
I think it would be ok, because it would happen only once for each person (on register, not log in)